src/calliope_pathways/models/italy_stationary/pre_processing/parse_lombardi.py

```diff
@@ -8,7 +8,7 @@
 from scipy.special import gamma
```

Comment:
If this is your only reference to scipy, you could remove this gamma function in favour of `numpy.random.gamma`, thereby reducing our overall dependencies

Reply:
I went for `math.gamma` instead. `scipy.special.gamma` is the gamma function, while `numpy.random.gamma` is the gamma distribution (which uses the gamma function).

Still, `scipy` dependency removed!